File: Java/1405.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int arr[] = new int[n];
		for(int i = 0; i < n; i++){
			arr[i] = sc.nextInt();
		}
		
		for(int i = 0; i < n; i++){
			int k = i;
			for(int j = 0; j < n; j++){
				if(k==n) k = 0;
				System.out.printf("%d ", arr[k]);
				k++;
			}
			System.out.println();
		}
	}
}

File: C/1405.c
#include <stdio.h>

int main(){
	int n;
	scanf("%d", &n);
	int arr[n];
	for(int i = 0; i < n; i++){
		scanf("%d", &arr[i]);
	}
	
	for(int i = 0; i < n; i++){
		int k = i;
		for(int j = 0; j < n; j++){
			if(k==n) k = 0;
			printf("%d ", arr[k]);
			k++;
		}
		printf("\n");
	}
	return 0;
}